open_academy/models/open_academy_partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
```

Comment:
Please rename this file and its references. The standard is 
![image](https://user-images.githubusercontent.com/11479473/182205005-64414316-e298-49a4-a419-3c03825f0d58.png) and the model is `res.partner` so the file should be `res_partner.py`